ED/src/init/ed_params.f90

```diff
@@ -2443,7 +2452,12 @@ subroutine init_pft_alloc_params()
    rho(3)     = 0.71   ! 0.60
    rho(4)     = 0.90   ! 0.87
    rho(5)     = 0.20   ! Copied from C4 grass
-   rho(6:11)  = 0.00   ! Currently not used
```

Comment:
I like the idea of assigning actual numbers for wood density for temperate species for completion.  I think it's no longer the case, but if I remember correctly, in the past rho=0 was a special flag to tell not to use wood density.

Reply:
I did not see such a flag when grep rho throughout the code. Maybe it has already been updated?